src/Workspaces/Core/Portable/Workspace/Solution/Solution.cs

```diff
@@ -304,6 +304,20 @@ public Solution WithProjectOutputRefFilePath(ProjectId projectId, string outputR
             return new Solution(newState);
         }
 
+        /// <summary>
+        /// Creates a new solution instance with the project specified updated to have the default namespace.
+        /// </summary>
+        internal Solution WithProjectDefaultNamespace(ProjectId projectId, string defaultNamespace)
```

Comment:
i thought this was extracted into a PR that already went in...

Reply:
It did, [here](https://github.com/dotnet/roslyn/blob/dev16.0-preview2/src/Workspaces/Core/Portable/Workspace/Solution/Solution.cs#L310). Not sure if it's because the code hasn't moved to the branch this review-only PR targeting, or some github glitch.

Reply:
OK, seems the code finally flew into this branch which caused conflict. I will address them when rebase and retarget this PR to preview2 branch, once you signed off.